core/management/commands/export_projects.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from django.core.management import BaseCommand
+
+from core.importer.projects_exporter import ProjectsExporter
+
+JSON_SUFFIX = '.json'
+
+
+class Command(BaseCommand):
```

Comment:
there is a lot of common code on this three files. Many use inheritance or a common function instead